src/acquisition/covidcast/database.py

```diff
@@ -298,6 +298,23 @@ def compute_covidcast_meta(self, table_name='covidcast', use_index=True):
       ORDER BY
         `time_type` ASC,
         `geo_type` ASC
+      
+      UNION
+      SELECT
+        `time_type`,
+        `geo_type`.
+        MIN(`issue`) as `min_issue`
+      FROM 
+        `{table_name}`
+      WHERE
+        `source` = %s AND
+        `signal` = %s AND
+      GROUP BY
+        `time_type`,
+        `geo_type`
+      ORDER BY
+        `time_type` DESC,
+        `geo_type`  DESC
```

Comment:
why did you do these in descending order?

Reply:
Oh I think because with the DESC order the first date will be the min-issue date, and with ASC the first date will be the max-issue date. But they are same because they use min() and max() to find the date, I will change it back~